packages/harpguru-core/src/components/harp-face-fragment/hooks/use-harp-rows/use-harp-rows.tsx

```diff
@@ -28,10 +29,7 @@ export const useHarpRows = (xRange: XRange): HarpRows => {
   // TOOMANYRENDERS: This is going to produce more renders than necessary
   // when:
   // - the
-  const [activeHarpStrata] = useGlobal('activeHarpStrata')
-  const {
-    apparatus: { interactionMatrix: fullInteractionMatrix },
-  } = activeHarpStrata
+  const [fullInteractionMatrix] = useGlobal('activeInteractionMatrix')
```

Comment:
the "full"prefix is good and should be considered for more common use.

There isn't enough consistency with the usage of "active" & "viewable" & "full" yet. I full review would be useful. Possibly after the mock objects setup helpers have been made.